molgenis-data-elasticsearch/src/main/java/org/molgenis/data/elasticsearch/reindex/ReindexActionMetaData.java

```diff
@@ -0,0 +1,91 @@
+package org.molgenis.data.elasticsearch.reindex;
+
+import static org.molgenis.data.EntityMetaData.AttributeRole.ROLE_ID;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.molgenis.MolgenisFieldTypes;
+import org.molgenis.data.support.DefaultEntityMetaData;
+import org.molgenis.fieldtypes.EnumField;
+import org.molgenis.fieldtypes.TextField;
+
+public class ReindexActionMetaData extends DefaultEntityMetaData
+{
+	public static final String ENTITY_NAME = "reindex_action";
```

Comment:
The _ name scheme is not quite the same as our usual ReindexAction with attributes called reindexActionGroup and actionOrder. Is this on purpose?